Changelog.md

```diff
@@ -13,6 +13,7 @@ Bugfixes:
  * SMTChecker: Fix internal error when using an empty tuple in a conditional operator.
  * SMTChecker: Fix internal error when using bitwise operators with an array element as argument.
  * Standard JSON Interface: Fix ICE when the optimizer is disabled and an empty/blank string is used for ``optimizerSteps`` sequence.
+ * StaticAnalyzer: Only raise a compile time error for division and modulo by zero when it's between literals.
```

Comment:
Shouldn't we mention that the previous behavior was unintended? I mean, just to make people aware that the compiler will not raise a compile time error anymore for non-literals but only runtime errors.

Reply:
Rethinking about it, maybe that is too big for a changelog entry, so I guess we can just mention it in the release blog post. So I approved the PR ;)